BioForm/api/views.py

```diff
@@ -0,0 +1,225 @@
+import datetime
+import jwt
+from profiles.models import *
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.exceptions import AuthenticationFailed
+from .serializers import *
+
+def serialize_user(user):
+    return {
+        'username': user.username,
+        'email': user.email,
+        'date_of_birth': user.date_of_birth,
+    }
+
+@api_view(['POST'])
+def register(request):
```

Comment:
Use [`ViewSets`](https://www.django-rest-framework.org/api-guide/viewsets/) to create these APIs, you can also use [Class-based Views](https://www.django-rest-framework.org/api-guide/views/#class-based-views) where required but don't use function-based views.